lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -242,12 +242,13 @@
     }
   }
 
-  h1 {
+  .instructor-dashboard-title {
     @extend .top-header;
     display: inline-block;
     margin-bottom: ($baseline*.75);
     padding-bottom: 0;
     border-bottom: 0;
+    text-align: left;
```

Comment:
I still need to look into this.  I will take a look as soon as the sandbox is updated.